README.md

```diff
@@ -46,6 +46,27 @@ while running:
 c.close()
 ```
 
+**AvroProducer**
+```
+from confluent_kafka import Producer
+from confluent_kafka.avro.avro_producer import AvroProducer
+from confluent_kafka.avro.cached_schema_registry_client import CachedSchemaRegistryClient
+from confluent_kafka.avro.serializer import util
+from confluent_kafka.avro.serializer.message_serializer import MessageSerializer
+
+value_avro_schema = util.parse_schema_from_file('ValueSchema.avsc')
+key_avro_schema = util.parse_schema_from_file('KeySchema.avsc')
+value = {"name": "Value"}
+key = {"name": "Key"}
+
+producer = Producer({'bootstrap.servers': 'mybroker,mybroker2'})
+
+schema_registry_url = 'https://<host>:<port>'
+
+avroProducer = AvroProducer(producer, schema_registry_url, value_schema=value_avro_schema, key_schema=key_avro_schema)
```

Comment:
Either make the names `default_value_schema` and `default_key_schema` or switch to maps from topic to schema:

    default_key_schemas, default_value_schemas